lib/textWidthCalculator.js

```diff
@@ -31,7 +32,28 @@ class TextWidthCalculator {
 
     // measure the text
     const textMetrics = this.canvasContext.measureText(word);
-    const { width } = textMetrics;
+
+    // default width metric
+    let { width } = textMetrics;
+
+    // When measuring the x-direction of a piece of text,
+    // the sum of actualBoundingBoxLeft and actualBoundingBoxRight
+    // can be wider than the width of the inline box (width), due to
+    // slanted/italic fonts where characters overhang their advance width.
+    // ref: https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics
+
+    // It can therefore be useful to use the sum of actualBoundingBoxLeft
+    // and actualBoundingBoxRight as a more accurate way to get the
+    // absolute text width:
+    if (this.useExtendedMetrics) {
```

Comment:
a gente vai passar esse cara por usuário? Vendo que o suporte é relativamente recente
https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics/actualBoundingBoxLeft
onde a gente vai definir se `this.useExtendedMetrics` é true ou false?
Digo, o controle dependendo da versão do browser

Reply:
O usuário habilita passando `options.useExtendedMetrics` como `true` na chamada o `wrapTextRobust`. Coloquei essa nova option na documentação (README).

Reply:
sim, minha dúvida é no nosso lado mesmo. Eu imagino que para gente essa opção seja habilitada como true por default. Como você planeja no nosso lado passar como `false` nos casos que não tem suporte? Qual a implicação disso na conta?

Reply:
Ah sim, entendi seu ponto. Eu não havia observado essa possibilidade de ter que trabalhar como `false` dependendo do suporte do browser.

Como no nosso caso as tolerâncias vão variar se usarmos esta opção, acho que o ganho não vai compensar o trabalho. Melhor termos apenas uma forma de cálculo e fixar essa tolerância, ainda que ela seja um pouco maior.

Mas isso não invalida esse PR, podemos ter essa feature. Quando/se precisarmos, já está pronto.

Reply:
por mim, manda bala